formatting/filesWithCRLFEndings/main-blinky.c

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
Similar with this file, a file with a single CRLF should be enough to trigger the action right?